packages/logger-plugin/src/logger.plugin.ts

```diff
@@ -87,4 +88,10 @@ export class NgxsLoggerPlugin implements NgxsPlugin {
 
     return ms_ie;
   }
+
+  private _hasPayload(event: any) {
```

Comment:
use `hasPayload` instead `_hasPayload`

Reply:
Most of the core codebase is using the `_` prefix.

Reply:
I do not respect this approach, unfortunately and very upset that everywhere it is used

Why then did the access modifier come up?

Reply:
From the beginning ;)
https://github.com/ngxs/store/commit/372377ff20aac74e61750534bc8503cbf00d838d

Reply:
typescript private access modifier most prefer 

the modifier is usually enough to understand that the method is private, I write in Java and absolutely do not understand why people come up with all these tricks

Reply:
When reading code, it makes it easier to understand the access modifier from the name, instead of using the mouse to scroll up (in PRs) or to move the mouse over the method name (in the IDE).

Reply:
it was part of the original code design. If we want to make a design change we should do that as part of a separate task

Reply:
is the type of event really any?

Reply:
The `event` is the name of the second parameter in the `handle` method of the plugin.
This is the name given for the parameter `action` in the `NgxsPlugin` interface:
```ts
handle(state: any, action: any, next: NgxsNextPluginFn): any;
```